LayoutTests/fast/events/mouse-events-on-node-deletion.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE HTML>
+<script src="../../resources/js-test.js"></script>
+<style>
+.box {
+    position: absolute;
+    top: 20px;
+    left: 20px;
+    height: 100px;
+    width: 100px;
+}
+
+#outer {
+    background-color: grey;
+}
+
+#inner {
+    background-color: red;
+}
+</style>
+
+<div id="outer" class="box">
+  <div id="inner" class="box"></div>
+</div>
+
+<div id="console"></div>
+
+<script>
+
+description("Verifies that mouse events are not fired on deleted nodes");
+
+function eventLogger(event) {
+  if (event.eventPhase == Event.AT_TARGET)
+    debug(event.currentTarget.id + " received " + event.type);
+}
+
+function init() {
+  var outer = document.getElementById("outer");
+  var inner = document.getElementById("inner");
+
+  ["mouseenter", "mouseleave", "mouseover", "mouseout"].forEach(function(event){
+    outer.addEventListener(event, eventLogger);
+    inner.addEventListener(event, eventLogger);
+  });
+
+  outer.addEventListener("click", function () {
+    outer.removeChild(inner);
+    debug("removed inner");
+  });
+}
+
+init();
+
+if (window.eventSender) 
+{
+  eventSender.mouseMoveTo(30, 30);
+  eventSender.mouseMoveTo(130, 130);
+  eventSender.mouseDown();
+  eventSender.mouseUp();
+  eventSender.mouseMoveTo(30, 30);
+  eventSender.mouseMoveTo(0, 0);
+} else 
+{
+  debug("This test requires eventSender");
```

Comment:
We should add a manual instruction for users to run this test within a browser.

Reply:
Yes. See [LayoutTests/fast/events/remove-target-in-mouseup.html](https://github.com/WebKit/WebKit/blob/main/LayoutTests/fast/events/remove-target-in-mouseup.html) for an example.

Reply:
That doesn't tell us whether the test is passing or failing.